config.yaml

```diff
@@ -0,0 +1,6 @@
+paths:
+  path_pdf: 'data\pdf\plik.pdf'
+  path_modified_pdf:  'data\pdf\modified_plik.pdf'
+
+message_container_height: 5000
```

Comment:
We can add an explanation what are these varaibles. 
For message_container_height I set it as some big value, because it war required to fit the chat window into the placeholder.